recipes/AceEditorHalogenHooks/web/index.js

```diff
@@ -0,0 +1,5 @@
+"use strict";
```

Comment:
Can we get rid of 'use strict' here?

Reply:
Yep, it's not needed 👍

Reply:
[64c3836](https://github.com/JordanMartinez/purescript-cookbook/pull/306/commits/64c383647c2b200326b27e4a23d58c37812d04b8)